clio-dataaccess-model/src/main/scala/org/broadinstitute/clio/server/dataaccess/elasticsearch/ElasticsearchUtil.scala

```diff
@@ -56,5 +56,18 @@ object ElasticsearchUtil {
       */
     def unsafeGet[T: Decoder](key: String): T =
       json.hcursor.get[T](key).fold(throw _, identity)
+
+    def getAsString(key: String): String = {
+      json.asObject
+        .flatMap(_.apply(key))
+        .map {
+          case s if s.isString =>
+            s.asString.getOrElse("")
```

Comment:
Can avoid scattering `getOrElse`s throughout the cases by making this a `flatMap` (and the last case gets wrapped in a `Some`).